hackathon_site/registration/jinja2/registration/emails/waitlisted_email_body.html

```diff
@@ -0,0 +1,11 @@
+<p>Hello {{ user.first_name|striptags }},</p>
+
+<p>The {{hackathon_name}} team has reviewed your application, and due to both the high caliber and high volume of applications, we have decided to defer your application to the next round. You will hear back from us by {{final_review_response_date}}regarding a final application decision.</p>
+
+<p>By visiting your <a href="https://ieee.utoronto.ca/makeuoft/dashboard">dashboard</a>, you can join or make changes to your team, and we’ll re-consider your application along with your teammates (though due to limited spots remaining, note that we may not make the same decision on all teammates). </p>
```

Comment:
This URL shouldn't be hard coded. Django has [a function called `reverse`](https://docs.djangoproject.com/en/3.1/ref/urlresolvers/#django.urls.reverse) which can determine URLs based on view names. We need an extra step here of including the full domain as well.

Take a look at the [activation email template](https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/registration/jinja2/registration/emails/activation_email_body.html#L1), which does something similar (though more complex) at the very top.

In this case, you can do `{{ request.build_absolute_uri(url("event:dashboard")) }}` inline with the href. We'll just need to remember to pass the request in as a context when rendering this email.